group.py

```diff
@@ -2,4 +2,49 @@
 
 # Your code to go here...
 
-my_group =
+class person:
+
+    def __init__(self,name,age,job=''):
+        self.name = name
+        self.age = age
+        self.job = job
+        self.relations = {}
+
+    def relationship(self, person2, relationship):
+        self.relations[person2] = relationship
+
+def max_age(group):
+    ### Takes an input of a list of names in the class "person" and outputs the maximum age in the group ###
```

Comment:
For python when documenting a function you use `"""`
```suggestion
    """ Takes an input of a list of names in the class "person" and outputs the maximum age in the group """
```